File: bot-javascript/src/commands/bot.js
const { MessageEmbed } = require('discord.js');
const { formatarData } = require('../utils');

const execute = (bot, msg, args) => {
    const resposta = new MessageEmbed()
    .setColor('#3399ff')
    .setAuthor('🤖 Minhas informações')
    .setTitle('Fala meu paçero!')
    .setDescription('Olá eu sou o mais novo bot criado pelo Gustavo!')
    .setThumbnail(bot.user.displayAvatarURL())
    .addField('**Meu nick**', bot.user.username)
    .addField('**Meu ID**', bot.user.id)
    .addField('**Criado em**', formatarData('DD/MM/YYYY, às HH:mm:ss', bot.user.createdAt))
    .setFooter(`🛡 2021 © ${bot.user.username}.`)
    .setTimestamp();

    msg.channel.send(resposta);
};

module.exports = {
    nome: "bot",
    descricao: "Mostra algumas informações do bot",
    execute
};

File: bot-csharp/src/BotDiscord.Application/Commands/Bot.cs
using System.Threading.Tasks;
using BotDiscord.Application.Common;
using Discord;
using Discord.Commands;

namespace BotDiscord.Application.Commands
{
    public class Bot : CommandModuleBase
    {
        public override string CommandName => "bot";
        public override string CommandDescription => "Mostra algumas informações do bot";

        [Command("bot")]
        public async Task BotCommand()
        {
            var resposta = new EmbedBuilder()
                .WithColor(new Color(51, 153, 255))
                .WithAuthor("🤖 Minhas informações")
                .WithTitle("Fala meu paçero!")
                .WithDescription("Olá eu sou o mais novo bot criado pelo Gustavo!")
                .WithThumbnailUrl(Context.User.GetAvatarUrl() ?? Context.User.GetDefaultAvatarUrl())
                .AddField("**Meu nick**", Context.User.Username)
                .AddField("**Meu ID**", Context.User.Id)
                .AddField("**Criado em**", Context.User.CreatedAt.ToString("dd/MM/yyyy - HH:mm:ss"))
                .WithFooter($"🛡 2021 © {Context.User.Username}.")
                .WithCurrentTimestamp()
                .Build();

            var mensagemEnviada = await SendCustomMessageAsync(resposta);
        }
    }
}